packages/sdk/src/core/contextInternal.ts

```diff
@@ -305,7 +305,7 @@ export class ContextInternal {
 
 	public start() {
 		if (!this.interval) {
-			this.interval = setInterval(() => this.update(), 0);
+			this.interval = setInterval(() => this.update(), 20);
```

Comment:
>20 [](start = 52, length = 2)

How did we come up with this number?

Reply:
It's sort of arbitrary. I thought it would be quick enough for most purposes without overwhelming the CPU if checking for updates takes a while.

---
In reply to: [447885855](https://github.com/microsoft/mixed-reality-extension-sdk/pull/641#discussion_r447885855) [](ancestors = 447885855)

Reply:
Have we noticed a problem with the update loop of the server?

---
In reply to: [447921544](https://github.com/microsoft/mixed-reality-extension-sdk/pull/641#discussion_r447921544) [](ancestors = 447921544,447885855)